weblate/trans/tests/test_hooks.py

```diff
@@ -1562,15 +1643,45 @@ class AzureBackendTest(HookBackendTestCase):
     def test_ping(self):
         self.assert_hook('{"diagnostics": "ping"}', None)
 
-    def test_git(self):
+    def test_git_old(self):
         url = "https://f.visualstudio.com/c/_git/ATEST"
         self.assert_hook(
-            AZURE_PAYLOAD,
+            AZURE_PAYLOAD_OLD,
             {
                 "branch": "master",
                 "full_name": "ATEST",
                 "repo_url": url,
-                "repos": [url],
+                "repos": [
+                    "https://dev.azure.com/f/c/_git/ATEST",
+                    (
+                        "https://dev.azure.com/f/be9b3917-87e6-42a4-a549-2bc06a7a878f/"
+                        "_git/278d5cd2-584d-4b63-824a-2ba458937249"
+                    ),
+                    "git@ssh.dev.azure.com:v3/f/c/ATEST",
+                    "https://f.visualstudio.com/c/_git/ATEST",
+                    "f@vs-ssh.visualstudio.com:v3/f/c/ATEST",
+                ],
```

Comment:
This still generates single URL, so either your code doesn't do what you expected or the testcase change is wrong.

Reply:
Now it generates more URLs, but they do not match what is here, see the testcase failure: https://github.com/WeblateOrg/weblate/pull/4954/checks?check_run_id=1489377589

Reply:
Yeah I saw it already. I'm very sorry for creating a lot of commits that do not work but unfortunately I didn't manage to run the tests locally.

Reply:
Don't worry about that, but why have you checked "Lint and unit tests pass locally with my changes." in that case? :-)

There is a guidance for that in https://docs.weblate.org/en/latest/contributing/start.html, but it might be tricky to install indeed.

Reply:
Should the tests run when I run `./rundev.sh test --failfast weblate.trans`?
Or do I need to do something differently?

Reply:
Yes, that should work (though I have no experience with the dev Docker setup). You might want to run only the hook tests as the full testsuite will take quite some time: `weblate.trans.tests.test_hooks`

Reply:
Looking at `rundev.sh` source, you need to first do `./rundev.sh start` and then the tests can be executed. I'm just testing the workflow whether it actually works ;-)

Reply:
I've made some improvements to that in 2063025589fd14ce2661b08e018ed61f2eaefd8d, but it worked as expected.

Reply:
I finally managed to run tests locally which - obviously - helped a lot :-)